tests/storage/restore.go

```diff
@@ -40,14 +40,16 @@ const (
 	stoppingVM                = "Stopping VM"
 	creatingSnapshot          = "creating snapshot"
 
-	newVmName                     = "new-vm"
 	macAddressCloningPatchPattern = `{"op": "replace", "path": "/spec/template/spec/domain/devices/interfaces/0/macAddress", "value": "%s"}`
 )
 
-var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
+var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
```

Comment:
I think `deleteWebhook(webhook)` in `AfterEach()` can mess with `test_id:5258`.

Reply:
Thanks for the input! I'll try to tackle it after I figure out why `restore-` is getting appended with `""`